lib/slack.js

```diff
@@ -58,10 +58,10 @@ export default class SlackData extends EventEmitter {
       return this.retry();
     }
 
-    let total = users.length;
+    let total = users.length - 1;
```

Comment:
wouldn't it make more sense to filter `slackbot` from the array of users instead of just subtracting 1?